File: 0001 Code Chef/0003 Good Turn/implementation/0010 nodejs/good_turn.js
const readline = require('readline');


const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});



rl.question('',(input)=>{
    stringArray = input.split(" ");
    x = parseInt(stringArray[0]);
    y = parseInt(stringArray[1]);
    sumof = x + y
    if(sumof > 6){
        console.log("YES");
    }else{
        console.log("NO");
    }
    rl.close();
})

File: 0001 Code Chef/0003 Good Turn/implementation/0001 c/good_turn.c
#include <stdio.h>

int main(){
    int x,y,sumof;
    scanf("%d %d",&x,&y);
    sumof = x + y;
    if(sumof > 6){
        printf("YES\n");
    }else{
        printf("NO\n");
   }
    return 0;
}